nixos/modules/services/web-apps/opencloud.md

```diff
@@ -0,0 +1,110 @@
+# OpenCloud {#module-services-opencloud}
+
+[OpenCloud](https://opencloud.eu/en) is an open-source, modern file-sync and
+sharing platform. It is a fork of oCIS, a ground-up rewrite of the well-known
+PHP-based NextCloud server.
+
+The server setup can be automated using
+[services.opencloud](#opt-services.opencloud.enable).
+
+## Basic usage {#module-services-opencloud-basic-usage}
+
+OpenCloud is a golang application and does not require an HTTP server (such as
```

Comment:
We always recommend nginx to get acme certs easily and this text kinda reads a bit like bad advice.